lib/create.rb

```diff
@@ -75,6 +75,10 @@ def self.create
     genre_name = gets.chomp.strip
     Genre.new genre_name
   end
+
+  def self.create_from(genres)
+    genres.map { |json| Genre.from_json json }
+  end
```

Comment:
I didn't pay attention before. I think having from_json in the genre class violates S in SOLID. I would recommend using Storage::load for this